loadtest/control/simulcontroller/controller.go

```diff
@@ -290,7 +334,28 @@ func (c *SimulController) Run() {
 		close(c.stoppedChan)
 	}()
 
-	c.serverVersion, _ = c.user.Store().ServerVersion()
+	// Init controller's server version
+	serverVersionString, err := c.user.Store().ServerVersion()
+	if err != nil {
+		c.sendFailStatus("server version could not be retrieved")
+		return
+	}
+	serverVersion, err := control.ParseServerVersion(serverVersionString)
```

Comment:
I assume this is parsing nicely dev versions as well, right?

Reply:
Yes, covered by this test: https://github.com/mattermost/mattermost-load-test-ng/pull/565/files#diff-e8c6a9bbb75c6f053b0f1dc8704cdcbc2a40e88aa867934ab3523cebd3109409R257